#### Задача 1

Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм. Используя уровень значимости 0.05 и правостороннюю критическую область, проверить данную гипотезу при условии, что в выборке из 100 шариков средний диаметр оказался равным 17.5 мм, а дисперсия известна и равна 4 мм.

#### Ответ

In [1]:
import numpy as np

In [2]:
a = 17
n = 100
alpha = 0.05
m = 17.5
D = 4
sigma = np.sqrt(D/n)
sigma

0.2

In [3]:
(a, a + 2*sigma)

(17, 17.4)

Выбранный уровень значимости $\alpha = 0.05$ означает, что для проверки гипотезы о среднем значении a = 17 требуется, чтобы с вероятностью 95% выборочное среднее попадало в правосторонний интервал, который (по правилу 2-х сигм) равен $(a, a + 2 \cdot \sigma)$. 

То есть критическая область лежит правее точки $a + 2 \cdot \sigma $ и если выборочное среднее = 17.5 попадёт в критическую область, то основная гипотеза отвергается. 

Значение выборочного среднего = 17.5 попадает в критическую область правее точки $ a + 2 \cdot \sigma = 17.4 $ => гипотеза о том, что шарики для подшипников имеют средний диаметр 17 мм, при выбранном уровне значимости $\alpha = 0.05$ отвергается.

#### Задача 2

Известно, что генеральная совокупность распределена нормально со средним квадратическим отклонением 16. Найти доверительный интервал для оценки математического ожидания a с надежностью 0.95, если выборочное среднее равно 80, а объём равен 256.

#### Ответ

По правилу 2-х сигм доверительный интервал = среднее значение $ \frac{+}{-} \: 2 \cdot \dfrac {\sigma} {\sqrt{n}}$

In [4]:
std_ = 16
mean_ = 80
n = 256
(mean_ - 2*std_/np.sqrt(n), mean_ + 2*std_/np.sqrt(n))

(78.0, 82.0)

#### Задача 3

Продавец утверждает, что средний вес пачки печенья составляет 200 г. Из партии извлечена выборка из 10 пачек. Вес каждой пачки составляет:

202, 203, 199, 197, 195, 201, 200, 204, 194, 190

Известно, что их веса распределены нормально. Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%?

#### Ответ

In [5]:
sample = np.array([202, 203, 199, 197, 195, 201, 200, 204, 194, 190])
sample

array([202, 203, 199, 197, 195, 201, 200, 204, 194, 190])

In [6]:
m = sample.mean()
m

198.5

In [7]:
n = 10
sigma = np.sqrt(((sample - m)**2).sum()/(n-1))
sigma

4.453463071962462

Табличное значение коэффициента Стьюдента

In [8]:
t_int = 2.8214

Проверим, что коэфф.Стьюдента = 2.8214, соответствует вероятности = 99%

In [9]:
from scipy import stats
t = stats.t(n-1)
t.cdf(2.8214)

0.9899993808040004

Расчёт доверительного интервала

In [10]:
(m - t_int*sigma/np.sqrt(n), m + t_int*sigma/np.sqrt(n))

(194.5265978950862, 202.4734021049138)

Доверительный интервал, построенный с учётом доверительной вероятности 99%, = (194.5, 202.4), включает значение предполагаемого среднего значения а = 200 => утверждение продавца, что средний вес пачки = 200 гр, верно.